source/reference/system-requirements.rst

```diff
@@ -49,19 +52,19 @@ these features:
 
 * Instruction Set Extensions:
 
-  - Supplemental Streaming SIMD Extension 3 (Intel® SSSE3)
+  - Supplemental Streaming SIMD Extension 3 (SSSE3)
   - Intel® Streaming SIMD Extensions 4.1 (Intel® SSE 4.1)
   - Intel® Streaming SIMD Extensions 4.2 (Intel® SSE 4.2)
   - Carry-less Multiplication (PCLMUL)
 
 The following processor families have been verified to run |CL|:
 
-* Intel® Core™ Processor Family (2nd generation or greater)
+* Intel® Core™ processor family (2nd generation or greater)
 * Intel® Xeon® E3-xxxx processor
 * Intel® Xeon® E5-xxxx processor
 * Intel® Xeon® E7-xxxx processor
-* Intel® Atom® processor C Series
-* Intel® Atom® processor E Series
+* Intel Atom® processor C Series
```

Comment:
This appears to align with Marketing.
https://www.intel.com/content/www/us/en/products/processors/atom/c-series.html